Git.hub/Repository.cs

```diff
@@ -87,10 +86,28 @@ public IList<Branch> GetBranches()
             return _client.GetList<Branch>(request);
         }
 
+        /// <summary>
+        /// Retrieves the name of the default branch
+        /// </summary>
+        /// <returns>The name of the default branch</returns>
+        public string GetDefaultBranch()
+        {
+            RestRequest request = new RestRequest("/repos/{user}/{repo}");
+            request.AddUrlSegment("user", Owner.Login);
+            request.AddUrlSegment("repo", Name);
+
+            var repo = _client.Get<Repository>(request).Data;

```

Comment:
Should this called be wrapped in `try...catch`? If it fails, would it crash the app?

Reply:
To be honest, I followed the standard of the other methods in this class. Do you think I should change it?

Reply:
Try throwing an exception from here and see where it detrimentally affects the hosting app.
If it does, we definitely need `try...catch`. If not, we can leave it as is.